src/alchemlyb/workflows/abfe.py

```diff
@@ -441,7 +444,12 @@ def estimate(self, estimators=("MBAR", "BAR", "TI"), **kwargs):
         for estimator in estimators:
             if estimator == "MBAR":
                 self.logger.info("Run MBAR estimator.")
-                self.estimator[estimator] = MBAR(**kwargs).fit(u_nk)
+                self.logger.info(
+                    "By default, n_bootstraps=50 is used to estimate the "
+                    "MBAR error. Supply n_bootstraps=0 to use analytic error."
```

Comment:
We should not change the default without a deprecation. I'd issue a DeprecationWarning to state that the error estimate will change in release 2.3 to n_bootstraps=50. 

Until then, keep 
```python
kwargs.setdefault('n_bootstraps', 0)
```
to keep the old behavior.

Reply:
I'd also use a more informative status message (see other comment) that reports on n_bootstrap.

Use a DeprecationWarning for letting users know about coming changes.

Reply:
Yes, I think for this PR I will just add a DeprecationWarning without changing the behaviour.